src/stylesheets/components/_results.scss

```diff
@@ -9,4 +9,29 @@
     justify-content: center;
     flex-direction: column;
   }
-}
+
```

Comment:
This should be in a separate PR, it's not related to popup component.

Reply:
I removed this